src/schemes/uGenericScheme.ts

```diff
@@ -18,7 +19,7 @@ export interface IUGenericScheme {
 
 export interface IProposalCreateOptionsGS {
   callData?: string
-  value?: number
+  value?: BN | number | string
```

Comment:
this cannot be number

Reply:
Why not?

Reply:
because value can be bigger than number

Reply:
...in which case the calling application would already have crashed before getting into this code.  This code can handle any number that an app is able to send to it.

Reply:
could you please add a test which send Number.MAX_SAFE_INTEGER +1 as the value ?
MAX_SAFE_INTEGER  =. 2^53-1

Reply:
I can tell you what will happen in your proposed case with the code as I've written it.  The result will be that unexpected incorrect numbers will be sent to the contract (see the screenshot below).

There is nothing we can do to prevent the user from submitting numbers like this.  We also can't prevent them from submitting totally invalid string representations.  We could validate the values before submitting them to the contract, but note that we don't do that anywhere else where quantities are accepted, so why start here?  We leave it up to the application to validate user input.

![image](https://user-images.githubusercontent.com/1821666/96800817-3c43c180-13d4-11eb-9e19-a84bad5eb279.png)

Reply:
please submit your test. we should be able to support these kind of big values .
e.g sending 1 ether - 1000000000000000000 .

Reply:
I changed the existing test to use new BN('1000000000000000000')